Sourcery/Parsing/FileParser.swift

```diff
@@ -144,16 +144,16 @@ final class FileParser {
             case .enumelement:
                 return parseEnumCase(source)
             case .varInstance:
-                return parseVariable(source, containedInProtocol: containingIn is SourceryProtocol)
+                return parseVariable(source, containedIn: containingIn as? Type)
```

Comment:
To be honest I'm not sure. Maybe specifically in that context in `FileParser` it works, but for example:
```
private func processContainedDeclaration(_ declaration: Any, within containing: (declaration: Any, source: [String: SourceKitRepresentable])) {
        switch containing.declaration {
        case let containingType as Type:
            process(declaration: declaration, containedIn: containingType)
        case let containingMethod as SourceryMethod:
            process(declaration: declaration, containedIn: (containingMethod, containing.source))
        default: break
        }
    }
```
I think doesn't really fit. Thoughts?